### ISSUE TITLE
Not everything gets translated in .LNG files

### ISSUE THREAD

**Issue by aybe** [joncampbell123/dosbox-x#2700]
Like the following:

**Screenshots**

![dosbox-x_2021-07-21_21-10-18](https://user-images.githubusercontent.com/1537591/126545996-3b5dbd57-b6d6-485f-90cc-1b1df79a06b2.png)

**Comment by Wengier** [comment]
Yes, you are right that config file comments cannot be translated at this time (the Help screen shown in the screenshots are basically comments for config options). There are quite a lot of config options in DOSBox-X, and I think translating all of them may not be an easy task. Perhaps these can be addressed later.

**Comment by aybe** [comment]
I understand, that'll get done over time, eventually! Le 22 juil. 2021 09:31, Wengier ***@***.***> a écrit :
Yes, you are right that config file comments cannot be translated at this time (the Help screen shown in the screenshots are basically comments for config options). There are quite a lot of config options in DOSBox-X, and I think translating all of them may not be an easy task. Perhaps these can be addressed later.
